oap-server/mqe-rt/src/main/java/org/apache/skywalking/mqe/rt/operation/LogicalFunctionOp.java

```diff
@@ -57,4 +65,31 @@ private static ExpressionResult viewAsSeq(MQEParser.ExpressionListContext expres
         return firstResult;
     }
 
+    private static ExpressionResult isPresent(MQEParser.ExpressionListContext expressionListContext,
+                                              MQEParserBaseVisitor<ExpressionResult> visitor) {
+        boolean present = false;
+        for (MQEParser.ExpressionContext expContext : expressionListContext.expression()) {
+            final ExpressionResult result = visitor.visit(expContext);
+            // the expression data exist and contains not empty value
+            if (result != null) {
+                present = result.getResults().stream().filter(Objects::nonNull).anyMatch(r ->
+                    r.getValues().stream().anyMatch(v -> !v.isEmptyValue()));
+            }
+            if (present) {
+                break;
```

Comment:
If the expression: `is_present(not_exist_metric,exist_metric)`, then will return `1`, is that expected?

Reply:
Yes. If there is any metrics present, it should be return 1.

Reply:
Don't forget added to `metrics-query-expression.md`